tests/suites/hup/suite.js

```diff
@@ -103,6 +103,29 @@ const doHUP = async (that, test, mode, hostapp, target) => {
 const initDUT = async (that, test, target) => {
 	test.comment(`Initializing DUT for HUP test`);
 
+	const bootConfig = await imagefs.interact(that.context.get().os.image.path, 1, async (_fs) => {
+		try {
+			return await require('bluebird').promisify(_fs.readFile)('/config.txt');
```

Comment:
There's no need to await this promise if it's being returned

Reply:
http://hassansin.github.io/Why-return-await-Is-a-Bad-Idea

Reply:
Hmm, this was the recommended syntax and it's used this way in the CLI and etcher as well.

https://www.npmjs.com/package/balena-image-fs

Reply:
That readme would probably be good to update as well. You could skip the await to save a bit of time, and use `.catch()` on the returned promise to do something like:

```
return require('bluebird').promisify(_fs.readFile)('/config.txt').catch((err) => {
    return undefined;
});
```

It's faster, simpler, and subjectively more readable to me.